css/src/components/tabs.css

```diff
@@ -1,21 +1,92 @@
 /* Tabs. */
 .wpseotab {
-  display: none;
+	display: none;
 }
 
 .wpseotab.active {
-  display: block;
+	display: block;
 }
 
 .wpseotab p.expl {
-  margin-left: 6px;
+	margin-left: 6px;
 }
 
 .wpseotab .tab-block {
-  display: block;
-  margin: 30px 0;
+	display: block;
+	margin: 30px 0;
 }
 
 .wpseotab p.expl strong {
-  font-size: 115%;
+	font-size: 115%;
+}
+
+.yoast-tabs {
+	font-size: 16px;
+	border-bottom: 1px solid var(--color-border);
+	padding-left: 16px;
+	overflow: hidden;
+}
+
+.yoast-tabs__list {
+	display: flex;
+	flex-wrap: wrap;
+	align-items: flex-end;
+	margin-bottom: 0;
+	margin-top: 0;
+	padding-top: 3px;
+}
+
+.yoast-tabs__list-item {
+	position: relative;
+	background-color: var(--color-secondary);
+	margin-bottom: 0;
+	border-top: var(--border-default);
+	border-right: var(--border-default);
+	border-left: var(--border-default);
+	box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.15);
+	margin-left: -1px;
+}
+
+.yoast-tabs__list-item-link {
+	display: block;
+	color: var(--color-dark);
+	text-decoration: none;
+	padding: 12px 16px;
+}
+
+.yoast-tabs__list-item-link:hover {
+	color: var(--color-primary);
+}
+
+.yoast-tabs__list-item--active {
+	position: relative;
+	top: 1px;
+	background-color: var(--color-white);
+	z-index: 1;
+}
+
+.yoast-tabs__list-item--active .yoast-tabs__list-item-link {
+	color: var(--color-primary);
+	padding: 16px;
+}
+
+@media (max-width: 782px) {
```

Comment:
any reason why we choose this viewport size? (this fine tho, just curious about the reason for this size)

Reply:
I noticed this was one of the breakpoints used by WordPress.